package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fauxton",
-  "version": "1.2.9",
+  "version": "1.3.0",
```

Comment:
Please change back to 1.2.9 because traditionally we only bump this version when there's a new CouchDB release.